setup.cfg

```diff
@@ -51,42 +52,38 @@ install_requires =
     ruamel.yaml
     scipy
     xarray
-    Fiona
-    Shapely
-    grpcio
+python_requires = >=3.7
+package_dir =
+    = src
+zip_safe = False
+
+[options.packages.find]
+where = src
 
 [options.extras_require]
 dev =
-    # Test
+    black
+    build
+    bump2version
     deepdiff
+    ipython
+    isort
+    nbsphinx
+    pre-commit
+    pycodestyle
     pytest
     pytest-cov
     pytest-mypy
     pytest-runner
-    types-python-dateutil
-    # Linters
-    isort
-    prospector[with_pyroma,with_mypy]
-    pycodestyle
-    yapf
-    # Dependencies for documentation generation
-    nbsphinx
     recommonmark
     sphinx
     sphinx_rtd_theme
-    # ipython syntax highlighting is required in doc notebooks
-    ipython
-    # release
-    bump2version
-    build
     twine
+    types-python-dateutil
```

Comment:
Should we add nbqa and other new tools used by pre-commit to dev dependencies?

Reply:
The pre-commit hooks are running in there own virtual env so they do no need to be added to dev deps. 

Unless you want to run them outside pre-commit, but I dont think that is need because you can also do `pre-commit run black --files src/ewatercycle/util.py` to run black on a single file.